transitiveDeps.md

```diff
@@ -0,0 +1,94 @@
+
+The outputs below assume you're using `npm v8.1.2` with a lockfile, but we will provide instructions for Yarn as well.
+
+### Identifying the dependency tree
+
+First, let's see how `node-fetch` is pulled into our dependency tree.
+
+```bash
+keyvault main % npm ls node-fetch
+keyvault@1.0.0 /home/user/my-app
+└─┬ @azure/keyvault-keys@4.3.0
+  └─┬ @azure/core-http@2.2.3
+    └── node-fetch@2.6.6
+```
+
+> If you are using Yarn you can use `yarn why node-fetch` with similar results.
+
+It looks like `node-fetch`, a dependency of `@azure/core-http@2.2.3` is being pulled in _transitively_ via `@azure/keyvault-keys@4.3.0`.
+
+Because I use a lockfile, running `npm install` again will not help me here. But `npm` provides a few utilities that can.
+
+### Using `npm audit fix`
+
+First, it's possible that you were alerted to a security vulnerability thanks to `npm audit`. In this case, the simplest solution might be to run `npm audit fix`. Let's see what that looks like:
+
+```bash
+keyvault main % npm audit fix
+
+changed 1 package, and audited 51 packages in 421ms
+
+2 packages are looking for funding
+  run `npm fund` for details
+
+found 0 vulnerabilities
+keyvault main % npm ls node-fetch
+keyvault@1.0.0 /home/user/my-app
+└─┬ @azure/keyvault-keys@4.3.0
+  └─┬ @azure/core-http@2.2.3
+    └── node-fetch@2.6.7
+```
+
+As you can see node-fetch has been updated to 2.6.7, without having to wait for a new version of `@azure/core-http`.
+
+For more information on `npm audit` please refer to the [npm-audit documentation][npm-audit].
+
+> If you are using Yarn you can use [yarn npm audit][yarn-npm-audit] with similar results.
+
+### Using `npm update`
+
+Not all updates are due to security vulnerabilities. Sometimes you just want to update a transitive dependency without deleting your lockfile. In that case, `npm update node-fetch` can help. Let's see what that looks like:
+
+```bash
+keyvault main % npm update node-fetch
+
+changed 1 package, and audited 51 packages in 320ms
+
+2 packages are looking for funding
+  run `npm fund` for details
+
+found 0 vulnerabilities
+keyvault main % npm ls node-fetch
+keyvault@1.0.0 /home/user/my-app
+└─┬ @azure/keyvault-keys@4.3.0
+  └─┬ @azure/core-http@2.2.3
+    └── node-fetch@2.6.7
+```
+
+Awesome!
+
+> If you are using Yarn you can use `yarn upgrade node-fetch` with similar results.
+
+> Note: In some previous versions of `npm`, `npm update` would only update _top-level_ dependencies. If you're using npm 6.x for example you can provide the `--depth` argument to achieve similar results.
+
+
+For more information on `npm update` please refer to the [npm-update documentation][npm-update].
+
+
+### I read this guide and still have no idea what to do
+
+Feel free to file an issue and start a discussion, we're here to help and will try to do so to the best of our abilities!
```

Comment:
We can provide a link to file an issue in our repo, to make it easier.

```suggestion
Feel free to [file an issue][file-an-issue] and start a discussion, we're here to help and will try to do so to the best of our abilities!
```

Reply:
Fun fact round 2: you can have spaces in the names of links! So this can just be `[file an issue]`.